simphony_ui/couple_CFDDEM.py

```diff
@@ -214,29 +179,25 @@ def run_calc(output_path, mesh_name):
     dem_wrapper.SP_extension[liggghts.CUBAExtension.PAIR_POTENTIALS] = \
         ['repulsion', 'cohesion']
 
-    time_OF = 0.0
-    time_LGT = 0.0
-    time_drag = 0.0
-
-    if mode_OF is not "none":
+    if mode_cfd is not "none":
         # Generate cell list
         cellmat = {}
         index = {}
         gridsize = [chansize[0]/numgrid[0],
                     chansize[1]/numgrid[1],
                     chansize[2]/numgrid[2]]
 
-        for cell in mesh_wOF.iter_cells():
-            LLN = [chansize[0]*2, chansize[1]*2, chansize[2]*2]
+        for cell in mesh_cfd.iter_cells():
+            lln = [chansize[0]*2, chansize[1]*2, chansize[2]*2]
```

Comment:
try to see if you get an idea of what this lln is. seems to be some sort of 3d length. A more meaningful varname would be better, but I have no clue either. See if you find any hints around,